src/query/graphite/native/aggregation_functions.go

```diff
@@ -89,7 +90,33 @@ func maxSeries(ctx *common.Context, series multiplePathSpecs) (ts.SeriesList, er
 	return combineSeries(ctx, series, wrapPathExpr("maxSeries", ts.SeriesList(series)), ts.Max)
 }
 
-// divideSeries divides one series list by another series
+func divideSeriesHelper(ctx *common.Context, dividendSeries, divisorSeries ts.Series, metadata block.ResultMetadata) (*ts.Series, error) {
```

Comment:
Make `ts.Series` just `*ts.Series` so you can pass them directly by reference? De-referencing them then taking a ref locally means if they are modified in this method they won't be modified in the caller's view of the series, unlikely to be what we want.